### PR TITLE
chore: versions & add dependencies

### PR DESCRIPTION
**Pull request by luhc228**

### CODE REVIEW

File: extensions/iceworks-app/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 0.1.3
```
Comment (alvinhui) [inline extensions/iceworks-app/CHANGELOG.md:3]: 0.1.30

File: extensions/iceworks-project-creator/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 0.1.3
```
Comment (alvinhui) [inline extensions/iceworks-project-creator/CHANGELOG.md:3]: 0.1.4

**Comment by alvinhui** [comment]
reviewed

File: extensions/iceworks-project-creator/CHANGELOG.md (hunk 2 above)
Comment (luhc228) [reply extensions/iceworks-project-creator/CHANGELOG.md]: fixed

File: extensions/iceworks-app/CHANGELOG.md (hunk 1 above)
Comment (luhc228) [reply extensions/iceworks-app/CHANGELOG.md]: fixed

**Pull request merged**